webapp/src/Controller/BaseController.php

```diff
@@ -240,8 +241,17 @@ protected function commitDeleteEntity(
                 );
                 $entityManager->clear();
                 $entity = $entityManager->getRepository(Problem::class)->find($entity->getProbid());
+            } elseif ($entity instanceof Team) {
+                $users = $entity->getUsers();
```

Comment:
Maybe add a comment as to why we do this? But I think it would be better even if we do the eventlogservice->log() at the end with *all* ID's at once, which will mean we don't have to do this and the below change. Probably adding a `bool $eventlog = true` as argument to this method and setting it to false for multi delete should work.